gslib/commands/rpo.py

```diff
@@ -80,6 +81,21 @@
 _set_help_text = CreateHelpText(_SET_SYNOPSIS, _SET_DESCRIPTION)
 _get_help_text = CreateHelpText(_GET_SYNOPSIS, _GET_DESCRIPTION)
 
+GET_COMMAND = GcloudStorageMap(gcloud_command=[
```

Comment:
I saw another PR covering this. Do you have chain of PRs. That's gonna be tricky in git.

Reply:
Yes, I do, I'll have to rebase my first branch 😅